### DISCUSSION TITLE
push service-mipush

### DISCUSSION THREAD

**Discussion by wrm244** [community/community#69756] [Mobile]
### Select Topic Area

Product Feedback

### Body

Hello, I am an Android user who uses the GitHub app. I really like your product, it allows me to view and manage my code repositories anytime and anywhere. However, I also found a problem, that is, on Xiaomi phones that do not have the FCM module installed, the notification push of the GitHub app is not timely and stable, and sometimes I even miss important notifications. This is very inconvenient for me as a user who needs to pay attention to code updates and merge requests in time.

I understand that the reason for this problem may be because Xiaomi phones use their own push service framework (MiPush), instead of Google’s FCM service. Therefore, if the GitHub app only relies on the FCM service to push notifications, then there will be compatibility issues on Xiaomi phones. To solve this problem, I suggest that you consider using the MiPush push module to enhance the notification push effect of the GitHub app on Xiaomi phones.

![IMG_20231011_151609.jpg](https://github.com/community/community/assets/54385511/3d073006-0db1-4b1d-8357-123018b6c66b)

**Comment by elisealix22** [comment]
@wrm244 Thank you for this detailed report! I opened an issue for discussion within our Android team. We will definitely look into this. Please let us know if you have any other requests in the meantime!
